src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/ExternalUserRolesController.kt

```diff
@@ -130,6 +128,56 @@ class ExternalUserRolesController(
     externalUserRolesService.removeRoleByUserId(userId, role)
     log.info("Remove role succeeded for userId {} and role code {}", userId, role)
   }
+
+  @PreAuthorize("hasAnyRole('ROLE_MAINTAIN_OAUTH_USERS', 'ROLE_AUTH_GROUP_MANAGER')")
+  @GetMapping("{userId}/assignable-roles")
```

Comment:
I've added on these role s- but not sure if they're needed as we don't have any on external users.  Can remove at a later date if decide not